lib/elixir/lib/module.ex

```diff
@@ -550,18 +554,19 @@ defmodule Module do
 
   """
   @spec safe_concat(binary | atom, binary | atom) :: atom | no_return
-  def safe_concat(left, right) do
```

Comment:
This should be `:: atom`, no `no_return`, could you fix this as you are fixing the other specs?